docs/docs/get-started/basic-concepts.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Basic Concepts
+sidebar_label: Basic Concepts
+slug: '/basic-concepts'
+sidebar_position: 4
+---
+
+Package
+-----------
+
+A package defines the set up logic for a containerized backend. Packages can be accessed via Github locators.
+
+```bash 
+kurtosis run github.com/kurtosis-tech/basic-service-package
+```
+
+Packages can accept a set of parameters, defined by the package author, which enable the package consumer to modify their deployed backend at a high-level without needing to know how to configure each individual service.
+
+```bash
+kurtosis run github.com/kurtosis-tech/basic-service-package \
+  '{"service_a_count": 2, 
+    "service_b_count": 2, 
+    "service_c_count": 1,
+    "party_mode": true}'
+```
+
+Enclave
+-----------
+
+An enclave is a "walled garden" in which Kurtosis runs a containerized backend. Enclaves can run anywhere that you have a Docker engine or a Kubernetes cluster. The main purposes of an enclave are resource isolation and "garbage collection". With an "enclave remove" command (`kurtosis enclave rm`), you can destroy all of the resources used to set up your environment and leave nothing hanging around on your machine(s).
+
+Plan
```

Comment:
I would add a sentence here to take readers to the "Advanced Concepts" section for Plan if they want to learn more.

Reply:
taking the extreme brevity stance on this iteration, any text i can delete is best IMO so will leave it here for now and add it in if it adds confusion